kernel/src/main/java/org/kframework/compile/checks/CheckSortTopUniqueness.java

```diff
@@ -18,20 +18,7 @@
  *   syntax KList ::= A
  *   syntax Bag ::= A
  */
-public class CheckSortTopUniqueness {
-    private final Set<KEMException> errors;
-
-    private final Module module;
-
-    /**
-     * Check that the given module has no sort that has multiple top sorts.
-     * @param errors to be updated when violations occur. Multiple violations will be accumulated in `errors`.
-     * @param module to be checked.
-     */
-    public CheckSortTopUniqueness(Set<KEMException> errors, Module module) {
-        this.errors = errors;
-        this.module = module;
-    }
+public record CheckSortTopUniqueness(Set<KEMException> errors, Module module) {
```

Comment:
This comment disappeared.

Reply:
Will address!

Reply:
The comment on this constructor just duplicates the sentiment in the top-level record comment, so I'm happy with the state of the change as it is after a second look.